asv_bench/benchmarks/index_concat.py

```diff
@@ -0,0 +1,20 @@
+from .pandas_vb_common import *
+
+
+class Int64Indexing(object):
+    goal_time = 0.2
+
+    def setup(self):
```

Comment:
@toobaz to see what size is best (without being able to run asv), just do the timing yourself with  `%timeit`, and set the size so that it takes something in the range of a few milliseconds to few tens of milliseconds